Makefile

```diff
@@ -92,3 +95,47 @@ delete-awsfederatedaccountaccess: check-aws-account-id-env
 
 .PHONY: test-awsfederatedaccountaccess
 test-awsfederatedaccountaccess: check-aws-account-id-env create-account create-awsfederatedrole create-awsfederatedaccountaccess test-switch-role delete-awsfederatedaccountaccess delete-awsfederatedrole delete-account
+
+# Create CCS (BYOC) namespace
+.PHONY: create-ccs-namespace
+create-ccs-namespace:
+	# Create namespace
+	@oc process -p NAME=${CCS_NAMESPACE_NAME} -f hack/templates/namespace.tmpl | oc apply -f -
+
+# Delete CCS (BYOC) namespace
+.PHONY: delete-ccs-namespace
+delete-ccs-namespace:
+	# Create namespace
+	@oc process -p NAME=${CCS_NAMESPACE_NAME} -f hack/templates/namespace.tmpl | oc delete -f -
+
+# Create CCS (BYOC) Secret
+.PHONY: create-ccs-secret
+create-ccs-secret:
+	# Create CCS Secret
+	./hack/scripts/aws/rotate_iam_access_keys.sh -p osd-staging-2 -u osdCcsAdmin -a ${OSD_STAGING_2_AWS_ACCOUNT_ID} -o ccs-secret.yaml
+	@oc apply -f ccs-secret.yaml
+	# Wait for AWS to propogate IAM credentials
+	sleep ${SLEEP_INTERVAL}
+
+# Delete CCS (BYOC) Secret
+.PHONY: delete-ccs-secret
+delete-ccs-secret:
+	# Create CCS Secret
+	@oc delete -f ccs-secret.yaml
+	@rm ccs-secret.yaml
+
+.PHONY: create-ccs-accountclaim
+create-ccs-accountclaim:
+	# Create ccs accountclaim
+	@oc process -p CCS_ACCOUNT_ID=${OSD_STAGING_2_AWS_ACCOUNT_ID} -p NAME=${CCS_CLAIM_NAME} -p NAMESPACE=${CCS_NAMESPACE_NAME} -f hack/templates/aws_v1alpha1_ccs_accountclaim_cr.tmpl | oc apply -f -
+	# Wait for accountclaim to become ready
+	@while true; do STATUS=$$(oc get accountclaim ${CCS_CLAIM_NAME} -n ${CCS_NAMESPACE_NAME} -o json | jq -r '.status.state'); if [ "$$STATUS" == "Ready" ]; then break; elif [ "$$STATUS" == "Failed" ]; then echo "Account claim ${CCS_CLAIM_NAME} failed to create"; exit 1; fi; sleep 1; done
+
+.PHONY: delete-ccs-accountclaim
+delete-ccs-accountclaim:
+	# Delete ccs accountclaim
+	@oc process -p CCS_ACCOUNT_ID=${OSD_STAGING_2_AWS_ACCOUNT_ID} -p NAME=${CCS_CLAIM_NAME} -p NAMESPACE=${CCS_NAMESPACE_NAME} -f hack/templates/aws_v1alpha1_ccs_accountclaim_cr.tmpl | oc delete -f -
+
+# Test CCS
+.PHONY: test-ccs
+test-ccs: create-ccs-namespace create-ccs-secret create-ccs-accountclaim delete-ccs-accountclaim delete-ccs-secret delete-ccs-namespace
```

Comment:
This would benefit from using the `check-aws-account-id-env` target like we do for `test-awsfederatedaccountacccess`, to validate that the `OSD_STAGING_2_AWS_ACCOUNT_ID` variable is set before attempting to create things.

Reply:
Yeap already doing this https://github.com/openshift/aws-account-operator/pull/322/files#diff-b67911656ef5d18c4ae36cb6741b7965R28

Reply:
I'll just reuse that same check, good catch.